torch/csrc/jit/interned_strings.h

```diff
@@ -43,6 +46,8 @@ _(Caffe2ConvTranspose) \
 _(ConvTranspose) \
 _(is_test) \
 _(epsilon) \
+_(expand) \
+_(Expand) \
```

Comment:
Why do we also add Expand here?

Reply:
`expand` == ATen op. `Expand` == ONNX op (not actually, because ONNX doesn't actually have expand yet, but I'm faking it for now.)